test/data/metadata/keywords/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+contact: The ACCESS Oracle
+email: oracle@example.com
+created: 2018-01-01
+description: Description
+notes: Notes
+keywords:
+  - cosima
```

Comment:
Are keywords case-sensitive?

Reply:
Yes, I guess I haven't put anything in for that. I'm not sure if we should enforce this at the database level (constraint), ORM level (normalisation in the constructor) or as part of `update_metadata` (probably not this high up...)

Reply:
from a user perspective I think a case-insensitive search would be least confusing

Reply:
Whichever level you choose I think it should be transparent to the user. So case should be transformed to lower case rather than raising an exception.

So up to you really. At the DB level means it remains consistent even if the data is changed in some other way for whatever reason.

Reply:
Thanks both. I've tried to do this in the maximally-consistent way, so searches are case-insensitive and results are presented as lowercase.